website/content/docs/setup/quickstart/index.md

```diff
@@ -3,6 +3,55 @@ title: Quick Start
 weight: 10
 ---
 
-This guide lets you quickly evaluate {{< name >}}. For a custom setup follow the [Installation instructions](/docs/setup/install).
- 
+This guide lets you quickly evaluate {{< name >}} using the builtin sample connectors and data access module. Instaling {{< name >}} control plane includes: 
+- Installing Hashicorp Vault and connector as the credentials manager 
+- Installing ODPi Egeria lab and connectors as the data catalog
+- Installing Open Policy Agent (OPA) and connector as the policy manager.
+- Arrow-Flight data access module for reading Parquet and CSV datasets
+
+For a custom setup follow the [Installation instructions](/docs/setup/install)
+
+The following sample was verified with the following software versions:
+- Kubernetes: kind v0.10.0, OpenShift 4.3
+You will also need to have
+- git
+
+The following steps assume you have the following available:
+- Kubernetes cluster
+- Kubectl
+
+1.  Obtain a local copy of {{< name >}} repository
+    ```bash
+    git clone https://github.com/IBM/the-mesh-for-data.git
+    ```
+1.  Change to the root directory of the repository
+    ```bash
+    cd the-mesh-for-data
+    ```
+1. Set Mesh for Data namespace
+    ```bash
+    kubectl config set-context --current --namespace=m4d-system
+    ```
+1. Install {{< name >}} control plane including the built-in sample connectors and software:
+    - Credentials manager: Hashicorp Vault (aka Vault)
+    - Data catalog: ODPi Egeria (aka Egeria)
+    - Policy manager: Open Policy Agent (aka OPA)
+
+    Install on OpenShift:
+    ```bash
+    WITHOUT_OPENSHIFT=false ./hack/install.sh
+    ```
+    **Or**
+    
+    Install on Kind:
+    ```bash
+    ./hack/install.sh
+    ```
```

Comment:
tested on kind and openshift but we shouldn't say "install on kind". Consider:
```suggestion
    ```bash
    ./hack/install.sh
    ```
   OpenShift users need to run `WITHOUT_OPENSHIFT=false ./hack/install.sh` instead.
```